app/forms/search_form.rb

```diff
@@ -4,6 +4,11 @@ class SearchForm
   include ActiveRecord::AttributeAssignment
 
   Option = Struct.new(:value, :label)
+  ApplicationType = Struct.new(:id, :name)
```

Comment:
I feel like since all we need is a thing with a value and a label, we could reuse the `Option` struct that does exactly that rather than make another struct that does exactly the same thing with different words.